akka-actor/src/main/scala/akka/actor/ActorSystem.scala

```diff
@@ -140,7 +140,7 @@ object ActorSystem {
   def apply(name: String, config: Option[Config] = None, classLoader: Option[ClassLoader] = None, defaultExecutionContext: Option[ExecutionContext] = None): ActorSystem = {
     val cl = classLoader.getOrElse(findClassLoader())
     val appConfig = config.getOrElse(ConfigFactory.load(cl))
-    new ActorSystemImpl(name, appConfig, cl, defaultExecutionContext).start()
+    new ActorSystemImpl(name, appConfig, cl, defaultExecutionContext, None).start()
```

Comment:
So far I have removed access to the untyped system from `typed.ActorSystem`, I still need to add the runtime check before I can consider this concern fully addressed.